client/src/main/java/com/paypal/selion/internal/platform/grid/BrowserFlavors.java

```diff
@@ -23,12 +23,15 @@
  */
 public enum BrowserFlavors {
     FIREFOX("*firefox"),
+    FIREFOX_HEADLESS("*firefox-headless"),
```

Comment:
Have you considered making "HeadLess" primarily a global config option ?  To me it fits neatly as a global option right along side the "runLocally" config option etc and less so as a Browser "flavor".

Reply:
See PR #376